src/GraphQL.Tests/Bugs/Bug1772.cs

```diff
@@ -26,7 +26,6 @@ public async Task DocumentExecuter_works_for_valid_operation(string? operationNa
     }
 
     [Theory]
-    [InlineData("")]
```

Comment:
The code previously tested to ensure that passing `""` would throw an operation-not-found exception.